flink-end-to-end-tests/flink-batch-sql-test/src/test/java/org/apache/flink/sql/tests/BatchSQLTest.java

```diff
@@ -114,104 +129,24 @@ public void testBatchSQL(BatchShuffleMode shuffleMode, @TempDir Path tmpDir) thr
         jobClient.getJobExecutionResult().get();
 
         final String expected =
-                "1980,1970-01-01 00:00:00.0\n"
-                        + "1980,1970-01-01 00:00:20.0\n"
-                        + "1980,1970-01-01 00:00:40.0\n";
+                "1980,1970-01-01 00:00:00\n"
```

Comment:
Why is this change is required?

Reply:
Hello! 
Yeah, the previous `CsvTableSink` is deprecated, I substitute it with the `filesystem` connector.
Simply I had to edit the expected result for some formatting internal of timestamp.
I just removed `.0` at the end, the result is, after all, the exact same.

Reply:
Got it 👍